crates/ott-balancer-bin/src/service.rs

```diff
@@ -83,7 +84,26 @@ impl Service<Request<IncomingBody>> for BalancerService {
                     .join("\n");
                     mk_response(rendered)
                 }
-                "metrics" => mk_response("TODO: prometheus metrics".to_owned()),
+                "metrics" => {
+                    // Create a Counter.
+                    let counter_opts = Opts::new("test_counter", "test counter help");
+                    let counter = Counter::with_opts(counter_opts).unwrap();
+
+                    // Create a Registry and register Counter.
+                    let r = Registry::new();
```

Comment:
This will need to be global so we can register metrics from anywhere. I'm fine if you want to defer that to another PR.